include/sys/time.h

```diff
@@ -51,6 +59,20 @@ typedef struct bintime {
     .sec = 0, .frac = ((1ULL << 63) / (hz)) << 1                               \
   }
 
+static inline time_t tm2sec(tm_t t) {
+  time_t res = 0;
```

Comment:
Why not `tm_t *` instead of `tm_t` function argument? This structure is quite big and now it's copied by value.

Reply:
Stayed by the convention that convert function takes the argument in such a way, but for this structure, it will be a better idea to use a pointer.